src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,96 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private T[] storage = (T[]) new Object[DEFAULT_CAPACITY];
+    private int storageSize;
+
     @Override
     public void add(T value) {
-
+        checkStorageSize();
+        add(value, storageSize);
```

Comment:
```suggestion
        add(value, storageSize);
```

Reply:
the checkStorageSize() method does not need to be called here, if necessary, the add(value, sizeStorage) method will increase the size of the array